### PR TITLE
Deploy Win Bastion 2 as part of array

### PR DESCRIPTION
**Pull request by ianrelf**
- Remove unused code

### CODE REVIEW

File: instances.tf [hunk 1]
```diff
@@ -24,7 +24,7 @@ resource "aws_instance" "bastion_linux" {
 }
 
 resource "aws_instance" "win_bastions" {
-  count                       = var.namespace == "prod" ? "1" : "2" # normally 2 - for Win Bastion 1 & Win Bastion 2
+  count                       = var.namespace == "prod" ? "2" : "2" # normally 2 - for Win Bastion 1 & Win Bastion 2
```
Comment (hemantgoswami) [inline instances.tf:27]: Values are same so do we need a ternary operator? we can simply assign 2?

**Review by hemantgoswami — COMMENTED** [review]
There's just one comment otherwise looks good

File: instances.tf (hunk 1 above)
Comment (ianrelf) [reply instances.tf:27]: Good question.
You are correct, we do not _need_ the ternary operator but we often _do_ vary the number of servers in NotProd compared to Prod - so it is helpful to keep it there.

**Review by hemantgoswami — APPROVED** [review]
LGTM

**Pull request merged**